nixos/modules/services/web-apps/plausible.nix

```diff
@@ -248,6 +248,7 @@ in {
             # setup
             ${cfg.package}/createdb.sh
             ${cfg.package}/migrate.sh
+            ${cfg.package}/bin/plausible eval '(Plausible.Release.prepare() ; Plausible.Auth.create_user("${ADMIN_USER_NAME}", "${ADMIN_USER_EMAIL}", "${ADMIN_USER_PWD}"))'
```

Comment:
it definitely fails on ofborg as well, for some reason it just doesn't mark as failed check in github

```
Cannot nix-instantiate `plausible.passthru.tests' because:
error:
       error: undefined variable 'ADMIN_USER_NAME'
       at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/ofborg-evaluator-1/nixos/modules/services/web-apps/plausible.nix:251:109:
          250|             ${cfg.package}/migrate.sh
          251|             ${cfg.package}/bin/plausible eval '(Plausible.Release.prepare() ; Plausible.Auth.create_user("${ADMIN_USER_NAME}", "${ADMIN_USER_EMAIL}", "${ADMIN_USER_PWD}"))'
             |                                                                                                             ^
          252|             ${optionalString cfg.adminUser.activate ''
```